src/components/toggle/dark-mode-toggle.tsx

```diff
@@ -0,0 +1,20 @@
+import React, {useState} from 'react';
+import 'react-toggle/style.css';
+import Toggle from './toggle';
+import './dark-mode-toggle.less';
+
+function DarkModeToggle() {
+    const [darkmode, setDarkmode] = useState(false);
+
+    localStorage.setItem(String(document.body.classList.toggle('darkmode', darkmode)), 'true');
```

Comment:
hm, maybe this can be more elegant...what should be stored in localStorage?

Reply:
Yup, i need help with this.. I'm thinking that we need to store either lightmode (default) or darkmode if that is chosen